OpenEdXMobile/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -217,4 +217,10 @@ public static void renderHtml(@NonNull TextView textView, @NonNull String html)
             viewText.setSpan(spanObj, start, end, flags);
         }
     }
+
+    @ColorRes
+    private static int getAuthorTextColor(@NonNull IAuthorData authorData) {
```

Comment:
I don't think we need this method; the `edx_brand_gray_base` should already be the default text color, and you should just add a check for the author being non-anonymous before changing the text color to blue (`edx_brand_primary_base`).